src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,228 @@
+            return;
+        }
+        String command = line.substring(0, dividerPosition);
+        String task = line.substring(dividerPosition + 1);
+
+        if (command.equals("done")) {
+            markTaskAsDone(task);
+
+        } else if (command.equals("todo")) {
+            addToList(new Todo(task));
+
+        } else if (command.equals(("delete"))) {
+            deleteTask(task);
+
+        } else if (command.equals("deadline")) {
+            int byPosition = 0;
+
+            try {
+                byPosition = splitCommandAndTask(line, "/by");
+            } catch (DukeException e) {
+                System.out.println("Please follow the deadline format: deadline *task* /by *date*");
+                return;
+            }
+            String deadlineTask = line.substring(dividerPosition + 1, byPosition - 1);
+            String by = line.substring(byPosition + 4);
+            addToList(new Deadline(deadlineTask, by));
+
+        } else if (command.equals("event")) {
+            int atPosition = 0;
+
+            try {
+                atPosition = splitCommandAndTask(line, "/at");
+            } catch (DukeException e) {
+                System.out.println("Please follow the event format: event *task* /at *date*");
+                return;
+            }
+            String eventTask = line.substring(dividerPosition + 1, atPosition - 1);
+            String date = line.substring(atPosition + 4);
+            addToList(new Event(eventTask, date));
+
+        } else {
+            System.out.println("Whoops you may have typed a wrong command");
+        }
+    }
+
+    public static int splitCommandAndTask(String line, String keyword) throws DukeException {
+        if (!line.contains(keyword)) {
+            throw new DukeException();
+        }
+        return line.indexOf(keyword);
+    }
+
+    
+    public static void runBot(Scanner in, String line) throws DukeException {
+        while (!line.equals("bye")) {
+            line = in.nextLine();
+            switch (line) {
+            case "bye":
+                break;
+            case "list":
+                displayList();
+                break;
+            default:
+                updateList(line);
+                break;
+            }
+        }
+    }
+
+    public static void printGoodbye() {
+        System.out.println("Bye see you SOON!");
+    }
+
+    public static void printHello() {
+        System.out.println("Hey it's your favorite chatbot buddy!");
+        System.out.println("How can I assist you today?");
+    }
+
+
+    public static void main(String[] args) throws DukeException {
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+//        System.out.println("Hello from\n" + logo);
```

Comment:
Avoid leaving commented code in your codebase.
If you even need this fragment in the future, you can retrieve it using revision control
The danger of leaving these in is that it may permanently be left in there.